.planning/phases/06-testing-expansion/06-03-PLAN.md

```
@@ -0,0 +1,277 @@
+</task>
+
+<task type="auto">
+  <name>Task 2: Add empty data and schedule parsing edge case tests (TEST-06, TEST-07)</name>
+  <files>tests/test_device_handlers.py</files>
+  <action>
+Add 12 edge case tests for empty data and schedule parsing:
+
+**Empty data tests (TEST-06) - add to appropriate classes:**
+
+1. `test_process_device_info_zones_key_missing` (TestSprinklerHandlerDeviceInfo)
+   - response["data"]["device"] has no "zones" key
```

markdownlint-cli2 (0.20.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: ""device""

(MD052, reference-links-images)